planetiler-core/src/main/java/com/onthegomap/planetiler/FeatureCollector.java

```diff
@@ -240,6 +240,11 @@ public long getSourceId() {
       return sourceId;
     }
 
+    public Feature setSourceId(long sourceId) {
+      this.sourceId = sourceId;
+      return this;
+    }
+
```

Comment:
let's change this to `setId`

Reply:
should we deprecate `getSourceId` and add `getId` ?

Reply:
We _could_ but I don't see any references to `getSourceId` in the code, so probably safe to remove.

Reply:
where you usually interact with `FeatureCollector` you also have access to the `sourceFeature`, right? so might be redundant to have a `getSourceId` that delegate to the `FeatureCollector`'s `source` when you can call `source.id()`